tests/test_http.py

```
@@ -60,46 +61,50 @@
+    adapter = client.sess.adapters["https://"]
+    assert isinstance(adapter, HTTPAdapter)
+    assert adapter.max_retries.backoff_factor == client.backoff_factor
 
 
-def test_pool_connections(client: Client):
-    client.configure(
-        pool_connections=99,
-        pool_maxsize=99,
-    )
-    assert client.pool_connections == 99
+def test_configure_pool_maxsize(client: Client):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_configure_pool_maxsize`

Add return type annotation: `None`

(ANN201)